apis/core/hydra/csv-source.ttl

```diff
@@ -57,6 +57,6 @@ cc:CSVSource
           a
             code:EcmaScript ;
           code:link
-            <file:handlers/csv-source#delete> ;
+            <file:handlers/csv-source#remove> ;
```

Comment:
Somehow the compiler was not too happy with a function called delete in the handler...